Demo/Helpers.swift

```diff
@@ -36,3 +36,17 @@ extension IndexPath {
         }
     }
 }
+
+struct Helpers {
```

Comment:
Maybe we could call this something like `Settings` instead of `Helpers`? 🤔

Reply:
Then, perhaps, we could move the `lastSelectedIndexPath` in there as well.

Reply:
Settings sound something like preferences and this is just state. Maybe call it 	`State`?

Reply:
True, `State` sounds right 😁